src/extensionInstall.mts

```diff
@@ -0,0 +1,71 @@
+import electron, { ProgressReporter } from '@vscode/test-electron';
+import { spawnSync } from 'node:child_process';
+import { createRequire } from 'node:module';
+import path from 'node:path';
+
+const require = createRequire(import.meta.url);
+
+export async function installExtensions(
+  extensionDevelopmentPath: string | string[],
+  extensions: string[] = [],
+  installDependentExtensions: boolean = false,
+  codeVersion?: string,
+  desktopPlatform?: string,
+  reporter?: ProgressReporter,
+) {
+  const vscodePath = await electron.downloadAndUnzipVSCode(codeVersion, desktopPlatform, reporter);
+
+  const extensionDevelopmentPaths = Array.isArray(extensionDevelopmentPath)
+    ? extensionDevelopmentPath
+    : [extensionDevelopmentPath];
+
+  const extensionsToInstall = installDependentExtensions
+    ? mergeDependentExtensions(extensions, extensionDevelopmentPaths)
+    : extensions;
+
+  const [cli, ...cliArgs] = electron.resolveCliArgsFromVSCodeExecutablePath(vscodePath, {
+    platform: desktopPlatform,
+  });
+
+  for (const extension of extensionsToInstall) {
```

Comment:
We can abort early if there are no extensionsToInstall (no extension dependencies)

Reply:
Done 108c71d